app/views/new_inbox.slim

```diff
@@ -1,2 +1,2 @@
 .container
-  h5 This has not been implemented yet.
```

Comment:
I feel as though for now this can also use the 'inbox' view. A new inbox is just an inbox that is blank.

Reply:
Similarly, for **new-inbox**, I added some new test case in spec/test_data.rb
![image](https://user-images.githubusercontent.com/12498220/99902038-d493d580-2cf5-11eb-8341-ce3200c43460.png)

And make **new-inbox** and **inbox** reuse the same views.
Now the **new-inbox** looks like:
![image](https://user-images.githubusercontent.com/12498220/99902078-19b80780-2cf6-11eb-9a56-670caf8d905b.png)